.github/workflows/platform-pull-request.yml

```diff
@@ -12,22 +12,40 @@ jobs:
     name: Validate Conventional Commit title
     runs-on: ubuntu-latest
     steps:
-      - uses: amannn/action-semantic-pull-request@v5
+      - name: Check PR Conventional Commit title
+        uses: amannn/action-semantic-pull-request@v5
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           types: | # mirrors changelog-sections in the /release-please-config.json
-            fix
             feat
+            fix
             infra
+            ci
             docs
-            chore
-            build
```

Comment:
Do we not need the `build` designator?

Reply:
I wanted to reduce the number and don't feel its too different to `ci` unless I'm missing something?

Reply:
Ok that makes sense.